run-terraform.sh

```diff
@@ -1,18 +1,21 @@
 #!/bin/bash
 
-cd credentials
-. ./credentials.sh
-cd ..
-
-cd run-dir
-rm *.tf
-cd ..
+if [ ! -d "run-dir" ]; then
+  	mkdir run-dir
+else
+  	cd run-dir
+  	rm *.tf
+  	cd ..
+fi
 
 for DIR in terraform/*; do
```

Comment:
Same as above. Add a brief comment on what this block is doing.

Reply:
Done